packages/pluto/readme.md

```diff
@@ -75,13 +82,13 @@ See the [Spec Type](src/spec.type.ts) for more information.
 ```ts
 // simple.test.ts
 
-import { registerActions, addTest, Equals } from '@creately/pluto';
+const pluto = require('@creately/pluto');
```

Comment:
What's the error we're getting when using `import` ?

Reply:
Something along the lines of 'package not found' because this is installed globally

Reply:
typescript gives that error when the typings are not available (or it can't find them). There are ways to fix it but let's address them in a new issue.

Reply:
Yep that's what I figured too, which we can possibly fix by including the typings in the module. There seems to be a solution [here](https://stackoverflow.com/questions/55659710/how-to-make-an-npm-module-with-globally-accessible-types) which I'll try out later.